docs/index.rst

```diff
@@ -19,11 +19,11 @@ PyBaMM documentation
 **Version**: |version|
 
 .. **Download documentation**:
-.. `PDF Version <https://numpy.org/doc/stable/numpy-user.pdf>`_ |
+.. `PDF Version <https://numpy.org/doc/1.23/numpy-user.pdf>`_ |
```

Comment:
Why link to numpy's documentation at all? This looks like a copy-paste error that got introduced in 14abfe4b.